pallets/sto/src/benchmarking.rs

```diff
@@ -0,0 +1,246 @@
+#![cfg(feature = "runtime-benchmarks")]
+use crate::*;
+use frame_benchmarking::benchmarks;
+use frame_support::traits::Get;
+use frame_system::RawOrigin;
+use pallet_settlement::{
+    benchmarking::{add_transfer_manager, compliance_setup},
+    VenueDetails,
+};
+use polymesh_common_utilities::benchs::User;
+use polymesh_common_utilities::{asset::AssetType, benchs::UserBuilder};
```

Comment:
```suggestion
use polymesh_common_utilities::{asset::AssetType, benchs::user};
```
You could use `user::<T>("prefix", 0)` instead of `<UserBuilder<T>>::default().generate_did().build("prefix")`. Just makes it a little less verbose.